kratos/includes/condition.h

```diff
@@ -15,22 +15,16 @@
 #define  KRATOS_CONDITION_H_INCLUDED
 
 // System includes
-#include <string>
-#include <iostream>
-#include <sstream>
-#include <cstddef>
```

Comment:
This is included virtually anywhere, it may clean the include list but they will be include at some other level so there is no real need to remove them from here. 

In fact string and iostream are used by the condition (not sure about `sstream` and `cstddef`)